### PR TITLE
add: homepage

### PR DESCRIPTION
**Pull request by bhushannaikaroskar**
Hey please review my PR for homepage

### CODE REVIEW

**Review by archit7448 — APPROVED** [review]
- you can add image or netlify link for UI 
- you can remove commented codes in navbar.jsx
- For category  you should use backend .
- code looks good to merge

File: src/components/navbar/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                >
+                    <div className="badge-container flex flex-column">
+                        
+                    </div>
+                </NavLink>
+                <div className="badge-container account flex flex-column">
+                    <NavLink
+                        className="btn btn-link-secondary justify-content-start font-color-gray"
+                        to="/login"
+                        // onClick={()=>setSearchValue("")}
```
Comment (srishtigautam21) [inline src/components/navbar/Navbar.jsx:29]: Remove commented code

File: src/components/navbar/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                    </NavLink>
+
+                    {/* {auth.isAuthenticated && (
+                        <div className="account-modal">
+                            <Link className="btn btn-link-secondary" to="/profile"> Profile </Link>
+                            <button className="btn btn-link-secondary font-error" onClick={logout}>
+                                Logout
+                            </button>
+                        </div>
+                    )} */}
```
Comment (srishtigautam21) [inline src/components/navbar/Navbar.jsx:41]: Commented code

**Review by srishtigautam21 — APPROVED** [review]
The code looks good to merge
Just remove the commented code

**Comment by bhushannaikaroskar** [comment]
> * you can add image or netlify link for UI
> * you can remove commented codes in navbar.jsx
> * For category  you should use backend .
> * code looks good to merge

File: src/components/navbar/Navbar.jsx (hunk 2 above)
Comment (bhushannaikaroskar) [reply src/components/navbar/Navbar.jsx:41]: Yup kept those for reference will remove one functionality of navbar is done.

**Pull request merged**